config/routes.rb

```diff
@@ -1,5 +1,9 @@
 Rails.application.routes.draw do
 
+  get 'errors/not_found'
```

Comment:
@danielpaul The match routes make sure that the 404 and 500 files are called for any methods like GET, POST and etc.